docs/source/data-model.rst

```diff
@@ -2,9 +2,21 @@
 Data Model
 **********
 
-Bluesky's event-based data model supports complex, asynchronous data collection
-and enables sophisticated live, prompt, streaming, and *post-facto* data
-analysis.
+A primary design goal of bluesky is to enable better research by recording
+rich metadata alongside measured data for use in later analysis. Documents are
+how we do this.
+
+A *document* is our term for a Python dictionary with a schema created
```

Comment:
This language should be updated since the RunEngine is now not the only way to create documents. Something like:

> A *document* is our term for a Python dictionary with a schema. The bluesky RunEngine emits documents during plan execution.

We _might_ want to elaborate:

> There are tools for composing documents directly, which can be useful for ingesting data that was not acquired with bluesky into this data model.

But that might be too much detail for this section.

Reply:
I am inclined to leave out the elaboration.

Reply:
I agree that is too much for this context.

Reply:
Great. I lean that way as well, though I included it for consideration.